src/components/SearchBar/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { BiSearch } from 'react-icons/bi';
+
+import style from './index.module.scss';
+
+/*
+    To use this component, pass the following props:
+    
+    > onSearchSubmit      : Pass a function that will handle the search submit
+    
+    > onSearchValueChange : Pass a function that will handle everytime the search input value changes
+
+    > placeholder         : Pass a string value for the placeholder
+*/
+
+const SearchBar = ({ onSearchSubmit, onSearchValueChange, placeholder }) => {
```

Comment:
i think it is better to have the onSearchSubmit and onSearchValueChange to be inside this component. future devs working on this will then just extract the value/event